client/ladder/ladder.tsx

```diff
@@ -342,7 +428,7 @@ export function LadderTable(props: LadderTableProps) {
 
   const renderRow = useCallback(
     (index: number) => {
-      const player = playersRef.current?.get(index - 1)
+      const player = playersRef.current?.[index]
```

Comment:
huh, didn't know you could do this with that syntax, weird. (You can also do it with function calls, in case you didn't know)